ui/src/app/pages/login/account-creation/account-creation-options.tsx

```diff
@@ -75,6 +76,14 @@ export const AccountCreationOptions = {
     {label: `Not Hispanic or Latino`, value: Ethnicity.NOTHISPANIC},
     {label: ` Prefer not to answer`, value: Ethnicity.PREFERNOANSWER}
   ],
+  gender: [
+    {label: 'Man', value: Gender.MAN},
+    {label: 'Non-Binary', value: Gender.NONBINARY},
+    {label: 'Transgender', value: Gender.TRANSGENDER},
+    {label: 'Woman', value: Gender.WOMAN},
+    {label: 'None of these describe me', value: Gender.NONE},
+    {label: 'Prefer not to answer', value: SexAtBirth.PREFERNOANSWER}
```

Comment:
nit: this value should be moved to a higher level since it can be used for non-gender-related things. Also I'd use `UPPER_SNAKE_CASE`.